pkg/virt-operator/creation/rbac/operator.go

```diff
@@ -231,6 +231,18 @@ func NewOperatorClusterRole() *rbacv1.ClusterRole {
 				},
 				Resources: []string{
 					"validatingwebhookconfigurations",
+					"mutatingwebhookconfigurations",
```

Comment:
I'm not sure that this should be a ClusterRole... can it be just a role in the deployment namespace?

Reply:
I don't think it changes anything regarding to security and we would have to maintain an extra role for the virt-operator service account.

Reply:
It does allow kubevirt to create `mutatingwebhookconfigurations` across the entire cluster unnecessarily. Though it's not a security issue by it's own, some cluster admin may wonder why this is needed.

With that said, since we currently do not maintain a role, I agree that it shouldn't be part of this PR

Reply:
webhook configurations are not namespaced.

Reply:
:+1: good point. :smile: 
so forget my comment.